mkt/features/utils.py

```diff
@@ -11,7 +11,8 @@ def load_feature_profile(request):
     if hasattr(request, 'feature_profile'):
         return
     profile = None
-    if request.GET.get('dev') in ('firefoxos', 'android'):
+    if request.GET.get('dev') in ('firefoxos', 'firefoxos+mobile',
+                                  'firefoxos+tv', 'android'):
```

Comment:
This list should be a constant instead now that it's becoming quite complex. Also, not sure we want feature detection on TVs  (does that even work for TVs?)